package.json

```diff
@@ -3,9 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^3.9.2",
     "axios": "^0.18.0",
     "g": "^2.0.1",
     "lodash": "^4.17.11",
+    "material-design-icons": "^3.0.1",
     "material-ui": "^0.20.2",
     "materialize-css": "^1.0.0-rc.2",
```

Comment:
Probably best to remove both `material-css` and `material-ui` . This will break the design, so no rush. I rather focus on cleaning and enhancing the logic in the coming days.